lib/Mojolicious/Plugin/CGI.pm

```diff
@@ -133,6 +133,7 @@ sub emulate_environment {
     HTTP_HOST         => $headers->host || '',
     HTTP_REFERER      => $headers->referrer || '',
     HTTP_USER_AGENT   => $headers->user_agent || '',
+    HTTP_IF_NONE_MATCH=> $headers->if_none_match || '',
```

Comment:
You need to run the commit through perltidy. There's a .perltidyrc in the project root with appropriate settings.